src/app/Http/Requests/AccountInfoRequest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Backpack\Base\app\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Validation\Rule;
+
+class AccountInfoRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        // only allow updates if the user is logged in
+        return \Auth::check();
```

Comment:
`use Illuminate\Support\Facades\Auth` makes this much clearer.

Reply:
Or auth()->check() if you don't like importing too much.